packages/block-directory/src/index.js

```diff
@@ -2,5 +2,4 @@
  * Internal dependencies
  */
 import './store';
-
-export { default as DownloadableBlocksPanel } from './components/downloadable-blocks-panel';
```

Comment:
It's a breaking change but we left a note in the block editor package:

> This package is meant to be used only with WordPress core. Feel free to use it in your own project but please keep in mind that it might never get fully documented.

Should we add a note about a breaking change?

Reply:
I still consider this package as `0.1` so, I don't really think it's that important at the moment.